x-pack/legacy/plugins/watcher/public/np_ready/application/models/action/webhook_action.js

```diff
@@ -25,7 +25,7 @@ export class WebhookAction extends BaseAction {
     this.username = get(props, 'username');
     this.password = get(props, 'password');
     this.contentType = get(props, 'contentType');
-
```

Comment:
Can you revert this whitespace change?

Reply:
@alisonelizabeth done

Reply:
It still looks like a change was made to this file (line removed). It's not doing any harm, but it would be preferred to revert it since this PR is unrelated to the watcher code.